news/_posts/2021-12-14-prestashop-beyond-1-7.md

```diff
@@ -0,0 +1,101 @@
+
+In that sense, PrestaShop 8 will continue and accelerate the transformative work towards the Future Architecture started in 1.7, but with a more pragmatic and focused approach (which will be covered in a dedicated article).
+
+This new version will focus on two main initiatives:
+
+##### 1. Technology update & cleanup
+
+In addition to updated libraries (including Guzzle 7) and an upgraded **Symfony 4.4** framework, PrestaShop 8 will introduce **compatibility with the newest PHP 8** version. We will also take the opportunity to remove old, deprecated code that had been replaced by newer components in 1.7.
+
+##### 2. Renewed product page
+
+The all-new Back office product page that was introduced as experimental in 1.7.8.0 will be fully integrated into PrestaShop 8. This new page will provide merchants with an enhanced user experience and increased performance, especially when dealing with a large number of combinations. Developers will also benefit from a fully Symfony-based page, with extendable forms and reusable Commands and Queries.
+
+### A company neutral project
+
+PrestaShop is a community open source project, giving anyone the freedom to download it, use it, and modify it to their needs. During the last few years, a lot of work has been done to clarify how an open source project like PrestaShop is different from a "freemium product".
+
+The maintainer team is committed to strengthening the community-built nature of the project. Last year, the project welcomed its first two maintainers not affiliated with PrestaShop SA. In 2021, to better distinguish the project from the company who originally created it, the project got a website of its own: [prestashop-project.org](https://www.prestashop-project.org/).
+
+In 2022, the differentiation between the open source project and the homonymous company will be pushed even further, by removing all company-related products and services from the project: PrestaShop 8 will no longer be bundled with the PrestaShop Addons marketplace, nor will it automatically download products from PrestaShop SA, nor will it depend on resources loaded from prestashop.com APIs. This represents a big leap forward in the project's neutrality, as the work on an independent project governance progresses.
+
+Of course, PrestaShop SA will continue to provide its products and services, for previous and new versions — but they will be distributed differently, as separate packages. These will likely be bundled in future products, which much like Linux distributions, will remain independent from the original open source project.
+
+Rest assured that contrary to other companies, who have chosen to abandon their open source platforms by treating them as spinoffs of their proprietary developments, PrestaShop SA will continue (and increase) its support to the open source project, and its future products will continue to be built _on top_ of the project — and not the other way around.
```

Comment:
contrary
or
contrarily

Reply:
Also this part seems quite grim 😛  almost like "I anticipate you will think Y so I tell you now that Y is false"

This could even backfire as _us saying this will not happen can make people believe it'll happen_ (because we lie! because we're evil, everybody knows that)

So I'm not sure about that paragraph 😞  .

Reply:
As far as I know, "contrary" is the right way for this formulation.

I'm not anticipating a backlash, I'm answering to people who have already asked me this question ;)

Also, it's a wink towards our competitors :P

> This could even backfire as us saying this will not happen can make people believe it'll happen (because we lie! because we're evil, everybody knows that)

Nothing that we ever say or do will make haters become constructive, so to me it makes no sense to tiptoe around their toxicness. The best we can do is ignore them and direct our energy towards the constructive ones.